packages/gatsby-theme-newrelic/src/components/NewRelicLogo.js

```diff
@@ -4,15 +4,21 @@ import { css } from '@emotion/react';
 import styled from '@emotion/styled';
 
 const Letter = styled.path`
-  fill: var(--heading-text-color);
+  fill: var(--text-color);
 `;
 
-const NewRelicLogo = ({ className, size, omitText }) => (
+const NewRelicLogo = ({
+  className,
+  size,
+  omitText,
+  textColor = 'var(--heading-text-color)',
```

Comment:
I think this is a similar situation where we could make use of the `css` property provided by emotion to do this in a "css"-y way.